setup.sh

```diff
@@ -42,7 +48,7 @@ setup() {
         profiles=""
         # Grab other profiles and concatenate in string, using comma as delimiter
         for entry in "/etc/netctl"/*; do
-            if [ -f "$entry" ]; then
+            if [ -f "$entry" ] && [[ ! "$entry" =~ ^$work_profile$ ]] ; then
```

Comment:
Just a sanity-check; does this actually work?

Try renaming your work profile to some string that doesn't contain 'work' as a substring.

Reply:
It works when testing externally, yes. f2dd1f9 fixes your suggestion, I tried my best to make the contents of the vars clear from their names, despite not containing 'work'.